src/index.js

```diff
@@ -0,0 +1,28 @@
+import navbar from './navbar';
+import home from './home';
+
+
+navbar();
+home();
+
+
+/* window.scroll(function(){
+    if (document.scrollTop() > 50){
+        var logo_place = document.getElementsByClassName("logo");
+        var navb = document.getElementsByClassName('nav')
+        navb.addClass('back_nav');
+        logo_place.remooveClass('logo').addClass('logo_short')
+    } else{
+        var logo_place = document.getElementsByClassName("logo_short");
+        logo_place.remooveClass('logo_short').addClass('logo');
+        navb.remooveClass('back_nav');
+    }
+});
+var navTrigger = document.getElementsByClassName('navTrigger')
+navTrigger.click(function () {
+
+    this.toggleClass('active');
+    document.getElementById("mainListDiv").toggleClass("show_list");
+    document.getElementById("mainListDiv").fadeIn();
+
+}); */
```

Comment:
Please remove commented out code. 
As can be seen from [this screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)
It is better to delete commented code. If you need to review a previous version of your code, use git history. 🙂